src/account-info/AccountDetails.js

```diff
@@ -64,14 +65,16 @@ class AccountDetails extends PureComponent<Props> {
           <PresenceStatusIndicator email={user.email} hideIfOffline={false} />
         </View>
         {userStatusText !== undefined && (
-          <RawLabel style={[styles.largerText, componentStyles.statusText]} text={userStatusText} />
+          <RawLabel style={componentStyles.statusText} text={userStatusText} />
         )}
+        <RawLabel style={componentStyles.statusText} text={user.email} />
+        {user.is_bot === true && <RawLabel style={componentStyles.statusText} text="BOT" />}
```

Comment:
The "BOT" string should be translated. Use a `Label` (found in `commons`) and add the "BOT" string to `static/translations/messages_en.json`. But this is moot because of the strategy I mentioned earlier.